schemas/gltf-asset.schema.json

```diff
@@ -13,6 +13,14 @@
         "type": "Asset type should be a string."
       }
     },
+    "bodyType": {
+      "description": "The bodyType of asset.",
+      "type": "string",
+      "default": "custom",
```

Comment:
It is a bit confusing that the default value "custom" for bodyType does not trigger the custom asset schema (assetCustom), but the regular asset types. Which schema should the default value trigger?